plugin/to-github.vim

```diff
@@ -79,8 +79,8 @@ function! ToGithub(count, line1, line2, ...)
     return 'Too many arguments'
   endif
 
-  " Get the branch and path, and form the complete url.
-  let branch = s:run('git symbolic-ref --short HEAD')
+  " Get the commit and path, and form the complete url.
+  let branch = s:run('git show-ref --hash HEAD')
```

Comment:
@shioyama I'll merge now and take care of it since I already have a few other changes in queue :)